src/main/java/core/basesyntax/service/OperationService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface OperationService {
+    void manageStorageCells(String operationCode, String fruitName, int amount);

```

Comment:
use FruitTransaction or FruitData in the method parameters
```suggestion
    void manageStorageCells(FruitData fruitData);

```